src/mgr/ActivePyModules.cc

```diff
@@ -12,27 +12,26 @@
  */
 
 // Include this first to get python headers earlier
-#include "Gil.h"
```

Comment:
@Naveenaidu - note that clang-format moved this include, which should not be moved (see comment above)

Reply:
thank you! fixed it now